src/sentry/api/serializers/models/organization.py

```diff
@@ -123,8 +123,6 @@ def serialize(self, obj, attrs, user):
             feature_list.append('suggested-commits')
         if features.has('organizations:new-teams', obj, actor=user):
             feature_list.append('new-teams')
-        if features.has('organizations:code-owners', obj, actor=user):
```

Comment:
I don't think this is as big of a deal, but I gotta point it out.

Since the older frontend code will rely on this, it means users will get this feature disabled until they hard refresh the page to get new frontend code.

If this is a concern, I'd just hard `feature_list.append('code-owners')`, then follow up in a bit for deletion.

Reply:
Hey good point, I'd forgotten about that. Although I don't think it's a big deal that some people potentially won't see the button if they like... switch orgs without doing a browser refresh or something. @MaxBittker wdyt?

Reply:
imo not a huge concern, but definitely good to know for broader features